.github/workflows/tests.yml

```diff
@@ -152,6 +152,27 @@ jobs:
           if [[ $cs_fix_files ]]; then
             bin/php-cs-fixer fix --config=.php-cs-fixer.php -v --dry-run --using-cache=no --show-progress=dots --diff $cs_fix_files
           fi
+        elif [[ "${{ matrix.commands }}" == "scaffolded files mismatch" ]]; then
+          wget -q -O /tmp/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && chmod 755 /tmp/jq
+          /tmp/jq -r '.extra["mautic-scaffold"]["file-mapping"] | to_entries[] | "diff -q \(.key | sub("\\[(project|web)-root\\]";".")) app/\(.value)"' app/composer.json > diff_commands.sh
+          bash diff_commands.sh | tee /tmp/diff_command_output.txt
+          rm diff_commands.sh
+          if [[ $(wc -l </tmp/diff_command_output.txt) -ge 1 ]]; then
+            echo "some scaffolded files were not updated"
+            echo "Please apply the same changes in the files mentioned above"
```

Comment:
Is there some command the dev can run to apply the changes? Or tell them somehow int he error message how to fix the problem?

Reply:
the files are mentioned in the output
(example of the current situation)

```
Files ./package-lock.json and app/assets/scaffold/files/package-lock.json differ
Files ./.gitignore and app/assets/scaffold/files/example.gitignore differ
Files ./upgrade.php and app/assets/scaffold/files/upgrade.php differ
```

I could add some line like 
```
please ensure the changes you made in the file(s) mentioned above are also present in their linked file in the `app/assets/scaffold/files` folder
```
I could parse that output and make example `cp` commands, but that's a bit too far IMO.
As some people only use an IDE and not CLI

Reply:
Let's step back. I missed the reason behind the app/assets/scaffold folder, so I'm the great example of a person who has no idea what's going on. I don't have to play dumb but just myself :)

Why do we have 3 files that must be the same in 2 different locations? Can it be done with symlinks?

Reply:
Those files are needed for the composer based flow for Mautic, as they live outside of `mautic/core-lib`  
I assume the decision for copies of the files was discussed and made when setting up the composer flow.  

e.g. Drupal has the same issue: both tarball based installs and composer based installs are supported.  
But the composer based install can't rely on the files around the application (e.g. .htaccess, index.php)

so those files are changes twice, in their normal location, and in a location that can be used for scaffolding.
See e.g. [this commit in Drupal core](https://git.drupalcode.org/project/drupal/-/commit/d0be0af780017fa7af8471a5c93c7c1aa4a114a7): 
it changes the .htaccess file in 2 locations.

there is an [ongoing issue about this in Drupal core](https://www.drupal.org/project/drupal/issues/3075954) to remove the duplicate ones, and only rely on the scaffolded folder

Reply:
I understand now. Thanks! The scaffolding check makes sense 👍 

I'll approve this PR once the CI is green.

Reply:
all tests green :)